exercise_1/say_hello_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "say_hello"
+require_relative 'say_hello'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.